pkg/alerts/lcid_expiration.go

```diff
@@ -0,0 +1,136 @@
+package alerts
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/google/uuid"
+	"go.uber.org/zap"
+
+	"github.com/cmsgov/easi-app/pkg/appcontext"
+	"github.com/cmsgov/easi-app/pkg/apperrors"
+	"github.com/cmsgov/easi-app/pkg/models"
+)
+
+func checkForLCIDExpiration(
+	ctx context.Context,
+	fetchSystemIntakes func(context.Context) (models.SystemIntakes, error),
+	updateSystemIntake func(context.Context, *models.SystemIntake) (*models.SystemIntake, error),
+	sendLCIDExpirationEmail func(ctx context.Context, requestName string, lcidExpirationDate *time.Time, intakeID uuid.UUID) error,
+) error {
+
+	// Fetch all system intakes
+	var result models.SystemIntakes
+	result, err := fetchSystemIntakes(ctx)
+
+	if err != nil {
+		appcontext.ZLogger(ctx).Warn(fmt.Sprintf("Failed to fetch system intakes for LCID Expiration check. %s", err))
+		return err
+	}
+
+	currentDate := time.Now()
+
+	for _, currIntake := range result {
+
+		// Check if intake has an LCID
+		if currIntake.LifecycleExpiresAt != nil {
```

Comment:
It might be worth converting some of these statements (at least this one) into early returns to reduce the indentation required, something like:

```go
if currIntake.LifecycleExpiresAt == nil {
    return nil
}
```

Reply:
Actually, my previous comment was wrong - it'd be `continue`, not `return` (as that would end the entire function early!)